boards/arm/mimxrt1064_evk/Kconfig.defconfig

```diff
@@ -10,6 +10,16 @@ if BOARD_MIMXRT1064_EVK
 config BOARD
 	default "mimxrt1064_evk"
 
+if CODE_INTERNAL_QSPI
+
+config NXP_IMX_RT_BOOT_HEADER
+	def_bool y
+
+config DEVICE_CONFIGURATION_DATA
+	def_bool n
```

Comment:
Why is this `def_bool n`?

Reply:
If it is set to y, although the dcd will be built into the header, but if there's a debug script to initialize sdram, it will fail to continue debug after programming.

Reply:
It's implicitly false already, so we don't need to set it false again here.